airflow/www/templates/airflow/gantt.html

```diff
@@ -23,6 +23,8 @@
     href="{{url_for('appbuilder.static',filename='datepicker/bootstrap-datepicker.css')}}" >
 <link type="text/css" href="{{ url_for('static', filename='css/gantt.css') }}" rel="stylesheet" />
 <link type="text/css" href="{{ url_for('static', filename='css/tree.css') }}" rel="stylesheet" />
+<meta name="dag-id" content="{{ dag.dag_id }}">
+<meta name="data" content="{{ data }}">
```

Comment:
This doesn't have `| tojson` on it - should it? Also how does this cope with a double-quote in the variable somewher?

Reply:
Quoting should be fine then, but I do think we need `tojson` on all of these (think, but an not certain)

Reply:
Now the quotation mark can not appear in the name of the dag.
If I tried to do it, I got an error message. 
>> airflow.exceptions.AirflowException: The key (example_bash_operator">Awesome) has to be made of alphanumeric characters, dashes, dots and underscores exclusively

I have changed the value of the dag_id field in a function from `views.py` file to show this problem.
 
<img width="711" alt="Screenshot 2019-04-23 at 13 47 51" src="https://user-images.githubusercontent.com/12058428/56578903-05b69c00-65cf-11e9-9724-625c5599540c.png">